build/Helix/runtests.cmd

```diff
@@ -78,7 +78,13 @@ FOR %%I in (WexLogFileOutput\*.jpg) DO (
 :SkipReruns
 
 cd scripts
+powershell -ExecutionPolicy Bypass .\OutputSubResultsJsonFiles.ps1 ..\te_original.wtl ..\te_rerun.wtl ..\te_rerun_multiple.wtl %testnameprefix%
```

Comment:
Optional: Consider having a single "post-process test results" script that does everything that is needed.
Right now we end up compiling the C# code twice, parsing the wtl files twice, etc.

Reply:
Having separate PowerShell files was useful for local debugging, since that way I could run only a single operation instead of needing to do the whole thing.  The amount of overhead involved is minimal in the grand scheme of things, so I think for that reason I'd prefer to leave things as is.